services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/DefaultFlowV2Spec.groovy

```diff
@@ -23,83 +24,6 @@ class DefaultFlowV2Spec extends HealthCheckSpecification {
     @Autowired
     Provider<TraffExamService> traffExamProvider
 
-    def "Systems allows to pass traffic via default and vlan flow when they are on the same port"() {
```

Comment:
why?

Reply:
we had the same test in `flowCrudV2Spec`